cardName.ts

```diff
@@ -0,0 +1,17 @@
+export enum CardName {
```

Comment:
* Lower case filenames only
* What does this even do? This isn't compiled (nor will I accept any code that requires complication), I don't understand what this would accomplish when published.